cluster/photon-controller/util.sh

```diff
@@ -0,0 +1,1088 @@
+#!/bin/bash
+
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# A library of helper functions that each provider hosting Kubernetes must implement to use cluster/kube-*.sh scripts.
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/../..
+# shellcheck source=./config-common.sh
+source "${KUBE_ROOT}/cluster/photon-controller/config-common.sh"
+# shellcheck source=./config-default.sh
+source "${KUBE_ROOT}/cluster/photon-controller/${KUBE_CONFIG_FILE-"config-default.sh"}"
+# shellcheck source=../common.sh
+source "${KUBE_ROOT}/cluster/common.sh"
+
+readonly PHOTON="photon -n"
```

Comment:
In this case it should be a string. It's used later to make a single string that is passed to a bash function.